### DISCUSSION TITLE
Ender 3 Max Neo Support

### DISCUSSION THREAD

**Discussion by SuperDodge** [mriscoc/Ender3V2S1#630] [General]
As a newbie trying to navigate which firmware is compatible with which of the 1,000,000 Reality models is really hard for me.  I wasn't around for the evolution of models so I don't understand which differences are markers of what models.

Is there a firmware here that is compatible with the new Ender 3 Max Neo I just bought?

**Comment by mriscoc** [comment]
Check:  https://github.com/mriscoc/Ender3V2S1/wiki#requirements

**Reply by Bukkza** [reply]
> Check: https://github.com/mriscoc/Ender3V2S1/wiki#requirements

Hello Miguel, I installed your firmware on my Ender3MaxNeo according to your instructions, and everything works. However, there's a catch: when the filament flow is set to 100%, it barely feeds at all. I have to manually change it to 350%. Can you help me what the problem is?

**Reply by mriscoc** [reply]
You need to calibrate your extruder steps https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Comment by mriscoc** [comment]
This can help to people with Ender 3 Max Neo printers: https://github.com/mriscoc/Ender3V2S1/discussions/519

**Comment by cprgolds** [comment]
Just a comment:  My Ender 3 Max Neo has a GD32F303 Arm chip and 4.2.2 board and does the Professional Software will not load.  

Marlin also does not officially support this chip.

Will be watching,,,,
